src/app/components/search/SearchResultsTable/ItemThumbnail.js

```diff
@@ -33,7 +34,9 @@ const ItemThumbnail = ({
     return (
       <div className={`${styles.thumbnail} ${styles.container}`}>
         <div className={styles.iconContainer}>
-          <MediaTypeDisplayIcon mediaType={mediaType} className={styles.mediaIcon} fontSize="var(--iconSizeDefault)" />
+          <Tooltip title={mediaType} arrow>
```

Comment:
I was not able to make this work, Dani... but in any case, we should translate this string, not show the `mediaType` in English.